### PR TITLE
Javascript-javascript3-week2

### PR DESCRIPTION
**Pull request by santhoshboinapally**

### CODE REVIEW

File: javascript/javascript3/week2/fetchWaiting.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//Fetching and waiting
+const url =
+    "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+
+function waitingTime() {
+    console.log("Please wait for 3 seconds.");
+    return new Promise(function(resolve) {
+        setTimeout(() => {
+            resolve();
```
Comment (florinmaticu) [inline javascript/javascript3/week2/fetchWaiting.js:9]: There is an error regarding the call to `resolve`. One argument needs to be passed to it.
`Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?ts(2794)`

File: javascript/javascript3/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        console.log("Blue circle moved");
+        await moveElement(greenBox, { x: 400, y: 20 });
+        console.log("green circle moved");
+    } catch (error) {
+        console.log(error);
+    }
+}
+translateOneByOne();
+
+async function translateAllAtOnce() {
```
Comment (florinmaticu) [inline javascript/javascript3/week2/main.js:15]: I have run the application and the balls don't seem to move to the expected circles.

Comment (santhoshboinapally) [reply javascript/javascript3/week2/main.js:21]: I think it will run now. Thankyou.

**Review by florinmaticu — APPROVED** [review]
Good job

**Pull request merged**